File: 8-kyu-kata/grasshopper-messi-goals/javascript/grasshopperMessiGoals.Test.js
// 8 kyu - Grasshopper - Messi Goals  [ ID: 55ca77fa094a2af31f00002a  (grasshopper-messi-goals) ]
// URL: https://www.codewars.com/kata/55ca77fa094a2af31f00002a
// Category: REFERENCE  |  Tags: FUNDAMENTALS
// *****************************************************************************
describe("total goals", function(){
    it ('should have the correct value', function () {
      Test.assertEquals(laLigaGoals, 43)
      Test.assertEquals(championsLeagueGoals, 10)
      Test.assertEquals(copaDelReyGoals, 5)
    })
    it ("should equal the total", function () {
      Test.assertEquals(totalGoals, 58)
    })
})

File: 8-kyu-kata/grasshopper-messi-goals/python/grasshopper_messi_goals_test.py
# 8 kyu - Grasshopper - Messi Goals  [ ID: 55ca77fa094a2af31f00002a  (grasshopper-messi-goals) ]
# URL: https://www.codewars.com/kata/55ca77fa094a2af31f00002a
# Category: REFERENCE  |  Tags: FUNDAMENTALS
# ******************************************************************************
import codewars_test as test


@test.describe("Fixed Tests")
def fixed_tests():
    @test.it("Basic Test Cases")
    def basic_test_cases():
        test.assert_equals(la_liga_goals, 43, "la_liga_goals should equal to 43")
        test.assert_equals(champions_league_goals, 10, "champions_league_goals should equal to 10")
        test.assert_equals(copa_del_rey_goals, 5, "copa_del_rey_goals should equal to 5")
        test.assert_equals(total_goals, 58, "total goals should equal to 58")
